### DISCUSSION TITLE
Iran GFW is detecting servers with xray core. How to evade detection?

### DISCUSSION THREAD

**Discussion by lostsoul6** [XTLS/Xray-core#3532] [General]
Hello Everyone ,

In Iran , we currently use tunneling to direct traffic from domestic VPS to a foreign VPS . Recently , GFW is detecting all v2ray servers and they are limiting access from outside to the domestic VPS . This renders the server useless for 90 days .

What I understood is that the tunneling method is not related to this issue . They are accurately detecting and banning servers with v2ray (xray core) on them.

Now my question is , how can I make it difficult for them to detect xray services? My configs are vless + tcp + headers . I saw people saying that I need to use vless + tls +tcp + headers . is that enough ? should I use nginx and fallback as well ?

Any other combinations or server configs that will make it harder for them to detect the existence of xray core and services on servers?

Thank you .

**Comment by devilofcyber** [comment]
try these 3 ways:
1=use vmess tcp 
2= install your panel on another system like:windows
3= try using sing box


(ولی داداش منم تونل کار میکنم ازین خبرا نی نمیفهمم منظورت چیه)

**Reply by lostsoul6** [reply]
@2745796340 

What tunnel method are you using?
How many users do you have?
Which data center is your Iran vps from?

The issues I have are for panels with more than 150 users on them .

**Reply by devilofcyber** [reply]
متودو که نمیتونم بگم،ولی نزدیک ۶۰۰ یوزره
دوتا پنل ۳۰۰ تایی

دیتاسنترم از اسیاتک میگیره روتش

**Comment by lostsoul6** [comment]
@2745796340 

تا به حال ایران اکسس نشدی ؟ خیلی عجیبه منم آسیاتک دارم . احتمالا کانفیگات پیچیده تر هست و vless tcp ساده نیست که نزدن هنوز

**Reply by devilofcyber** [reply]
اکسس شدم یه بار فقط اونم همین هفته پیش،
نه کانفیگام اینان:
vless tcp http
vmess tcp http

**Reply by lostsoul6** [reply]
احتمالا مصرف پورتتون کمه . زیر ۱۵۰ مگابیت معمولا نمیخوره یا کم میخوره . مصرف پورتتون چقدره و چند آنلاین دارید؟

**Reply by devilofcyber** [reply]
هر پنل مکسش ۳۰ مگابایته،میانگین ۲۰
انلاینی دقت نکردم
